docs/Acceptable-Casks.md

```diff
@@ -28,11 +28,11 @@ Stable versions live in the main repository at [Homebrew/homebrew-cask](https://
 
 ### But There Is No Stable Version!
 
-When an App is only available as beta, development, or unstable versions, or in cases where such a version is the general standard, then said version can go into the main repo.
+If upstream does not consider the software stable, the cask can likely wait until they do.
```

Comment:
This should make the rule clearer:

```suggestion
When software is only available as a beta, development, or unstable version, its cask can go in the main repo. After a stable version is out, further updates to unstable versions will no longer be accepted.
```

We should keep the bit about exceptions further down. E.g. if there’s a major security hole in a stable version that is fixed in a beta.

Reply:
But this goes against your argument of "often upstream recommends a beta"